cloudflare/resource_cloudflare_load_balancer_monitor.go

```diff
@@ -25,20 +25,38 @@ func resourceCloudflareLoadBalancerMonitor() *schema.Resource {
 
 		SchemaVersion: 0,
 		Schema: map[string]*schema.Schema{
-			"expected_body": {
+			//
```

Comment:
I don't think these comments are adding any value and we can do away with them. We should be able to use the schema (via `ValidateFunc` and `ConflictsWith`) itself to determine if we are providing the incorrect values.

Reply:
I have not been able to find a way to validate one parameter based on the content of another the parameter. For example, we don't want "expected_body" to be set if the content of "type" is "tcp".
That's why I put those comments in the schema.

But if you have a sample of code to check a parameter based on the content of another one or if you still think the comments are useless, I'll remove them.